server/src/db/migrations/20220503145737_createRestaurants.cjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("restaurants", (table) => {
+    table.bigIncrements("id")
+    table.string("name").notNullable()
+    table.string("location").notNullable()
+    table.timestamp("createdAt").notNullable().defaultTo(knex.fn.now())
+    table.timestamp("updatedAt").notNullable().defaultTo(knex.fn.now())
+  });
+};
+
+/**
+ * @param {Knex} knex
+ */
+exports.down = (knex) => {
+    return knex.schema.dropTableIfExists('restaurants')
```

Comment:
Make sure to be consistent about single or double quotations throughout the project. I would suggest talking as a group, choosing one or the other and going with that.